.github/workflows/check-pr.yaml

```diff
@@ -0,0 +1,47 @@
+name: check-pr
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Extract dart version
+      uses: blendfactory/dvm-config-action@1efe1a203d0a3a31bffd767b0affb251b54ff6bc # v2.0.0
+      id: dvm-config-action
+
+      - name: Set up dart
+        uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c # v1.7.1
+        with:
+          sdk: ${{ steps.dvm-config-action.outputs.dart-sdk-version }}
```

Comment:
[memo]
- Sets up the appropriate runtime environment
  - checkout
  - extract dart version
  - set up dart
- The hash value of the version is the same as the following description
https://github.com/yumemi-inc/flutter-yumemi-lints/blob/33b8b503d730e9d528404e973f6c5f800322990e/.github/workflows/push-tag.yaml#L25
https://github.com/yumemi-inc/flutter-yumemi-lints/blob/33b8b503d730e9d528404e973f6c5f800322990e/.github/workflows/update-lint-rules.yaml#L38
https://github.com/yumemi-inc/flutter-yumemi-lints/blob/33b8b503d730e9d528404e973f6c5f800322990e/.github/workflows/update-lint-rules.yaml#L44